checkstyle-tester/projects-for-travis.properties

```diff
@@ -5,5 +5,4 @@
 guava|git|https://github.com/google/guava|v28.2||
 # Few projects that delivers set of unusual Java constructions that shall be correctly handled by AST visitor
 # 'InputAllEscapedUnicodeCharacters' must be skipped because it is too big and slows down JXR
-# InputCyclomaticComplexityRecords.java - bug in checkstyle
 #checkstyle|git|https://github.com/checkstyle/checkstyle.git|master|**/.ci-temp/**/*,**/resources-noncompilable/**/asttreestringprinter/**/*,**/resources-noncompilable/**/filefilters/**/*,**/resources-noncompilable/**/main/**/*,**/resources-noncompilable/**/suppressionsstringprinter/**/*,**/resources-noncompilable/**/gui/**/*,**/resources-noncompilable/**/javadocpropertiesgenerator/**/*,src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParser.java,**/InputAllEscapedUnicodeCharacters.java,**/resources-noncompilable/**/javaparser/InputJavaParser.java,**/resources-noncompilable/**/checks/metrics/cyclomaticcomplexity/InputCyclomaticComplexityRecords.java,**/resources-noncompilable/**/checks/imports/unusedimports/InputUnusedImportsSingleWordPackage.java
```

Comment:
Remove this file reference from this line, at the end of long line.

Please do this in all other times

Reply:
Fixed